test/parallel/test-assert.js

```diff
@@ -409,6 +409,11 @@ assert.throws(() => {
 }, /Got unwanted exception: user message/,
               'a.doesNotThrow ignores user message');
 
+assert.throws(() => {
+  assert.doesNotThrow(makeBlock(thrower, Error), 'user message');
+}, /Got unwanted exception: user message\ntest/,
+              'a.doesNotThrow ignores message of the unwanted exception');
```

Comment:
I actually tried to align `'` with `/` but I was unable to pass linter in this case. It was saying I have wrong indentation. 

Since it is ok to remove third argument this is exactly what I did.